### ISSUE TITLE
Using "Tab" key in a keyboard shortcut did not work

### ISSUE THREAD

**Issue by mowestusa** [realh/roxterm#243]
First, thanks @realh for all of your work in making roxterm a great Linux terminal that is highly configurable. I got exposed to it through MX Linux 21, Fluxbox Edition that has roxterm as its default terminal program. I'm also using it on Fedora 35 Workstation.

In order to make my tabs mimic the way tabs work in most browsers, I attempted to edit the following entry:

`Tabs/Next Tab=<Control>Page_Down` to `Tabs/Next Tab=<Control>Tab`

In most browsers `Ctrl+Tab` cycles through the tabs in your browser. I was hoping to create a similar effect with the above keyboard shortcut and the option selected **Wrap when switching tabs**.

However, after saving the above change and even trying to restart Roxterm, the keyboard shortcut does not move to the next tab, it does nothing. I can learn the default keyboard shortcut if the "Tab" key is not allowed to be used in a keyboard shortcut. I just thought it might be neat to match the keyboard shortcuts of roxterm with browsers.

The above was tested on Fedora 35 with Roxterm version 3.11.1.

**Comment by realh** [comment]
It looks like this is a [long-standing issue](https://bugs.launchpad.net/ubuntu/+source/gnome-terminal/+bug/162000) with GTK not supporting Tab as a shortcut. But you may be able to use <Primary>Tab as a workaround.

**Comment by mowestusa** [comment]
Thank you @realh for such a quick response. I have tested this possible solution in both MX Linux Fluxbox Edition and on Fedora 35.

1. I tried editing the default configuration file for roxterm keyboard shortcuts presented in the **Configuration Manager** changing the value of `<Control>Page_Down` to `<Primary>Tab`. The shortcut failed to switch tabs in roxterm on both systems.
2. Now with that configuration file changed, I attempted to edit the setting on both systems using

`gsettings set org.gnome.Terminal.Legacy.Keybindings:/org/gnome/terminal/legacy/keybindings/ next-tab '<Primary>Tab'`

The shortcut failed to switch tabs in roxterm even after roxterm was closed and restarted.

3. Finally, I did test on Fedora 35 where I have Gnome terminal installed, and after I used the `gsettings` command I could cycle through tabs in Gnome terminal, but since I was trying to get this behavior in roxterm, this wasn't a good solution.
4. A possible solution: According to the link you provided, it appears that other terminal programs like Konsole have been able to enable the `Ctrl+Tab` shortcut to cycle through open tabs. Browsers that run on Gnome and other DE's and WM's have also be enabled the `Ctrl+Tab` shortcut which I have in both Firefox and Vivaldi the two browsers that I use on Linux. However, this must involve some patch that you would have to do as a programmer in your code to get around this bug in the GTK stack. I'm sure you have more important things to do, and I would probably prefer that you worked on some other issues.

I have decided that my fix will be to retrain my brain. Although, `Ctrl+Tab` seems to make sense, I have also discovered that both Firefox and Vivaldi allow you to move through your open tabs with `Ctrl+Page_Up` and `Ctrl+Page_Down`. So I will just learn those as my new default keyboard shortcuts to move through tabs in both my browsers and in roxterm.

I will consider this "closed" since it isn't really a bug in roxterm but in the GTK stack. Thanks again for your quick response and your excellent work on roxterm.
